src/ontology/mondo-ingest.Makefile

```diff
@@ -181,12 +182,25 @@ config/%_term_exclusions.txt config/%_exclusion_reasons.robot.template.tsv: conf
 	--component-signature-path $(word 4,$^) \
 	--config-path $(word 5,$^)
 
-# Combines all tables
+# Exclusions: combined
 config/term_exclusions.txt config/exclusion_reasons.robot.template.tsv: config/ordo_term_exclusions.txt config/doid_term_exclusions.txt config/omim_term_exclusions.txt config/icd10cm_term_exclusions.txt config/icd10who_term_exclusions.txt config/ncit_term_exclusions.txt
-	@rm config/term_exclusions.txt; rm config/exclusion_reasons.robot.template.tsv; \
-	cat config/*_term_exclusions.txt > config/term_exclusions.txt; \
+	cat config/*_term_exclusions.txt > config/term_exclusions.txt
 	awk '(NR == 1) || (NR == 2) || (FNR > 2)' config/*_exclusion_reasons.robot.template.tsv > config/exclusion_reasons.robot.template.tsv
 
+# Problematic exclusions: by ontology
+$(REPORTDIR)/%_excluded_terms_in_mondo_xrefs.tsv $(REPORTDIR)/%_excluded_terms_in_mondo_xrefs_summary.tsv: $(TMPDIR)/mondo.sssom.tsv tmp/mondo.owl metadata/%.yml $(REPORTDIR)/component_signature-%.tsv $(REPORTDIR)/mirror_signature-%.tsv python-install-dependencies
+	python3 $(RELEASEDIR)/src/analysis/problematic_exclusions.py \
+	--mondo-mappings-path $(TMPDIR)/mondo.sssom.tsv \
+	--onto-path $(TMPDIR)/component-download-$*.owl.owl \
+	--onto-config-path metadata/$*.yml \
+	--mirror-signature-path $(REPORTDIR)/mirror_signature-$*.tsv \
+	--component-signature-path $(REPORTDIR)/component_signature-$*.tsv \
+	--outpath $@
+
+# Problematic exclusions: combined
+$(REPORTDIR)/excluded_terms_in_mondo_xrefs.tsv: $(foreach n,$(ALL_COMPONENT_IDS), $(REPORTDIR)/$(n)_excluded_terms_in_mondo_xrefs.tsv)
```

Comment:
Target for combined results.

Reply:
Can you commit the table? I think its much easier for me to review inputs / outputs than code..

Reply:
@matentzn OK, I added an exception to `.gitignore`. Here's how it handles `reports/` currently. Should I make any other updates to have any other files committed, or just these are fine?:

```.gitignore
src/ontology/reports/*
!src/ontology/reports/README.md
!src/ontology/reports/excluded_terms_in_mondo_xrefs.tsv
```

Reply:
nope looks good